tests/provision/facts/test-guest.sh

```diff
@@ -40,7 +42,7 @@ rlJournalStart
             fi
 
         elif [ "$PROVISION_HOW" = "container" ]; then
-            provision_options="--image fedora:39"
+            provision_options="--image $TEST_IMAGE_PREFIX/fedora/39/unprivileged:latest"
             bfu_provision_options="$provision_options --user=nobody"
```

Comment:
`--user` would need to change as well, to `fedora`, and the main image shouldn't be the unprivileged one. Together, I think something like this should be the outcome:

```
provision_options="--image $TEST_IMAGE_PREFIX/fedora/39:latest"
bfu_provision_options="--image $TEST_IMAGE_PREFIX/fedora/39/unprivileged:latest --user=fedora"
```